strawberryfields/backends/base.py

```diff
@@ -237,14 +237,15 @@ def prepare_vacuum_state(self, mode):
         """
         raise NotImplementedError
 
-    def prepare_coherent_state(self, alpha, mode):
+    def prepare_coherent_state(self, r, phi, mode):
```

Comment:
Hi @ziofil, just leaving some preliminary comments :blush:

Perhaps this is something upcoming, the `TFBackend.prepare_coherent_state` will need an update as well right? Got an error for it when trying to run the machine learning tutorial (as part of the docs).

Reply:
I probably didn't catch all of the occurrences of alpha